student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+require './classroom'
+
+class Student < Person
+  def initialize(age, name = 'Unknown', parent_permission = 'unknown')
+    super(age, name, parent_permission)
+  end
+
+  attr_accessor :class_room
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+end
```

Comment:
- [ ] in order to make this work you will need to set up the 'belongs-to' side as well as mentioned in [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).